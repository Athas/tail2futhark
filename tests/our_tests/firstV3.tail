i2d(firstV{[int],[1]}(iotaV(5)))
