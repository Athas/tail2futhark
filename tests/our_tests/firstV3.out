fun real main() =
  toReal(iota(5)[0])